covid19-dashboard/calculate_data.py

```diff
@@ -18,6 +18,31 @@
 DateToValueListType = List[Tuple[str, float]]
 
 
+def _difference(date_to_value: DateToValueListType,
```

Comment:
please add documentation for the function

Reply:
fixed.